src/whylogs/core/datasetprofile.py

```diff
@@ -131,6 +131,7 @@ def __init__(
         self._metadata = metadata.copy()
         self.columns = columns
         
+        self._active=True
```

Comment:
Redundant? I don't see the usage anywhere.

Also, a profile shouldn't have active or not - active is tied to logger/session because they are coupled with a writer and log rotation -  but a profile is just a holder for data

Reply:
we can remove it for now, we do cache the profiles, but the idea of the active profile will be to keep the profile metadata alive when uncache the profile, so that the profile scould be reloaded.

Reply:
```suggestion

```